### ISSUE TITLE
entity selecting when viewer.scene3DOnly = true

### ISSUE THREAD

**Issue by anne-gropler** [AnalyticalGraphicsInc/cesium#3523]
Minimum code to reproduce this in Sandcastle:
```
var viewer = new Cesium.Viewer('cesiumContainer', {scene3DOnly:true});

viewer.camera.setView({
    destination :Cesium. Cartesian3.fromDegrees(13.305, 52.528, 3000)
});

viewer.entities.add({
    position: Cesium.Cartesian3.fromDegrees(13.305, 52.528, 3000),
    cylinder : {
    length : 400.0,
    topRadius : 0.0,
    bottomRadius : 200.0
    }
});
```
Scroll out to see the cone. Click on it to select it. The green brackets should follow the cone, but they don't.
Double-click to track it. You will see the green brackets, the rest is black. Click the home button to untrack it and it will throws:
```
RangeError: Invalid array length
RangeError: Invalid array length
    at updateFrustums (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:1083:36)
    at createPotentiallyVisibleSet (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:1248:13)
    at render (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:1931:9)
    at Scene.render (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:1968:13)
    at CesiumWidget.render (https://cesiumjs.org/Cesium/Source/Widgets/CesiumWidget/CesiumWidget.js:675:25)
    at render (https://cesiumjs.org/Cesium/Source/Widgets/CesiumWidget/CesiumWidget.js:70:32)
```

This does not happen with `scene3DOnly: false`.
This does not happen when you add an Ellipse instead of a cylinder.
And this does not happen when you add 2 entities. Just with one.
Strange: Adding one cylinder from within in the viewer (call it A), and others from within a viewerMixin (call it B), this happens with BOTH. So add one in A: bug, and add one in B: bug, although there are 2 entities now in the same entitiesCollection. Add one more in A: bug fixed for A's entities, but not for B's entity.

**Comment by hpinkos** [comment]
Thanks for the very detailed description @anne-gropler!  This is definitely a strange bug.  We'll try take a look at it soon!

**Comment by hpinkos** [comment]
Hi @anne-gropler,  This was fixed with #4461 and will be available in the 1.27 release coming out on November 1st.  Let me know if you are still experiencing this problem, thanks!
